docs/ocis/release_notes.md

```diff
@@ -0,0 +1,282 @@
+
+See the [Documentation](<https://owncloud.github.io/ocis/deployment/owncloud10_with_oc_web/>) on how to deploy Infinite Scale in bridge mode.
+
+{{< hint "warning" >}}
+**Technology Preview**
+
+ownCloud Infinite Scale is currently in Technology Preview. The bridge mode should only be used in non-production environments.
+{{< /hint >}}
+
+### What to expect?
+
+This is the first promoted public release of ownCloud Infinite Scale, released as "Technical Preview". Infinite Scale is not yet ready for production installations. Technical audiences will be able to get a good understanding of the potential of ownCloud's new platform.
+
+Version 1.0.0 comes with the base functionality for sync and share with a much higher performance-, stability- and security-level compared to all available platforms. Based on ten years of experience in enterprise sync and share and a long standing collaboration with the biggest global science organizations this new platform will exceed what content collaboration is today.
+
+### How to get started?
+
+One of the most important objectives for oCIS was to ease the setup of a working instance dramatically. Since oCIS is built on Google's powerful GO language it supports the single-file-deployment: Installing oCIS 1.0.0 is as easy as downloading a single file, applying execution permission to it and get started. No more fiddling around with complicated LAMP stacks.
+
+#### Deployment Options
+
+Given the Golang-based architecture of oCIS, there are various deployment options based on the users requirements. With our experience with the for many users difficult setup of the LAMP stack before a big emphasis was put on easy yet functional deployment strategies.
+
+{{< tabs "deployments" >}}
+{{< tab "Single binary" >}}
+#### Delivery as single binary
+
+The single binary is the best option to test the new ownCloud Infinite Scale 1.0.0 Technical Preview release on a local machine. Follow these instructions to get the platform running in the most simple way:
+
+1. Download the binary
+
+    **Linux**
+    `curl https://download.owncloud.com/ocis/ocis/testing/ocis-testing-linux-amd64 --output ocis`
+
+    **MacOS**
+    `curl https://download.owncloud.com/ocis/ocis/testing/ocis-testing-darwin-amd64 --output ocis`
+
+2. Make it executable
+   `chmod +x ocis`
+
+3. Run it
+   `./ocis server`
+
+4. Navigate to <http://localhost:9200> and log in to ownCloud Web (admin:admin)
+
+oCIS environments to last should use a more sophisticated setup, see <https://owncloud.github.io/ocis/deployment/> for more information.
+{{< /tab >}}
+{{< tab "Docker" >}}
+#### Containerized Setup
+
+For more sophisticated and production setups we recommend using one of our proposed docker setups. See the [documentation](https://owncloud.github.io/ocis/deployment/ocis_traefik/) for a setup with Traefik as a reverse proxy which also includes automated SSL certificate provisioning using Letsencrypt tools.
```

Comment:
We say that this is a tech preview and it shouldn't be used in production but here we offer documentation for production setups?